libafl_frida/src/lib.rs

```diff
@@ -451,9 +462,16 @@ mod tests {
             );
 
             {
+                
+                #[cfg(target_os = "linux")]
```

Comment:
Why do we need the `libloading::os::unix::Symbol`?

Reply:
We need to use the unix version of it specifically because we need to use `RTLD_NOW` when opening the library. Without `RTLD_NOW`, it will not resolve the symbols in the lib for you. 

This is problematic mainly because of the plt handler. Because the symbol still needs to be resolved, when the hook is checked in the callout it will not detect anything to hook (Because it's still pointed to the PLT resolver stub). This will cause the tests to fail. So, using `RTLD_NOW` is a quick fix for this.